go/vt/vtgate/planbuilder/builder.go

```diff
@@ -117,8 +118,8 @@ type columnOriginator interface {
 // VSchema defines the interface for this package to fetch
 // info about tables.
 type VSchema interface {
-	FindTable(tablename sqlparser.TableName) (*vindexes.Table, error)
-	FindTableOrVindex(tablename sqlparser.TableName) (*vindexes.Table, vindexes.Vindex, error)
+	FindTable(tablename sqlparser.TableName) (*vindexes.Table, key.DestinationTarget, error)
```

Comment:
Let's rename this to VCursor, because the `FindTable` function signature doesn't match the one in VSchema any more.